src/fs.js

```diff
@@ -45,7 +51,23 @@ function makeDirs(dirs) {
 	});
 }
 
+/** Delete files from a given path */
+function deleteFiles(dirs, options = {}) {
```

Comment:
It's not `dirs` anymore, I’d call it `patterns` because `dir` accepts globs.

Reply:
Default arguments aren’t supported in Node 4, that’s why the CI is failing.

Reply:
I should checked the `engines` key 😅